server/api/albums.js

```diff
@@ -31,25 +31,18 @@ albumsRouter.get('/:id', async (req, res, next) => {
   }
 });
 
+//ALBUMS EDITING ROUTES FOR ADMINS - ADD, DELETE, EDIT
+
 albumsRouter.post('/', async (req, res, next) => {
   try {
-    const {
-      title,
-      description,
-      genre,
-      year,
-      price,
-      photoURL,
-      quantity,
-    } = req.body;
     const newAlbum = await Album.create({
-      title,
-      description,
-      genre,
-      year,
-      price,
-      photoURL,
-      quantity,
+      title: req.body.title,
```

Comment:
i think we use the destructure from req.body format elsewhere, but i'm not super opposed to this format

Reply:
I think I usually do the destructure to use those variables elsewhere.

But yeah, whatever version people like is fine by me, just needs to be consistent. Just be careful modifying all these, be sure those files aren't in current edits before you change them :)